alpha/scripts/utils/updateEntriesRecordings.php

```diff
@@ -0,0 +1,105 @@
+<?php
+require_once('/opt/kaltura/app/tests/lib/KalturaClient.php');
+if ($argc < 6)
+{
+	die("Usage: php $argv[0] partnerId adminSecret serviceUrl simuliveEntryIdsFile vodEntryIdsFile \n");
+}
+$partnerId = $argv[1];
+$adminSecret = $argv[2];
+$url = $argv[3];
+$simuliveEntryIdsFile = $argv[4] ;
+$vodEntryIdsFile = $argv[5];
+
+$config = new KalturaConfiguration();
+$config->serviceUrl = $url;
+$client = new KalturaClient($config);
+$ks = $client->session->start(
+	$adminSecret,
+	'',
+	KalturaSessionType::ADMIN,
+	$partnerId,
+	86400,
+	'disableentitlement');
+$client->setKS($ks);
+
+$simuliveEntryIds = file ($simuliveEntryIdsFile) or die ('Could not read file'."\n");

```

Comment:
@ravitshalem  why read two different files? will be much simpler to have one CSV file ( i.e. <live entry ID>,<vod entry ID>)